### PR TITLE
feat(app-file-manager): add file thumbnail and action config

### PR DESCRIPTION
**Pull request by Pavel910**
## Changes
This PR removes the old `FileManagerFileTypePlugin` in favor of the config API. This allows us to have more control over individual actions and thumbnails (in Grid and File Details independently).

With this PR we also register Webiny Version widget (in the main navigation) as a regular menu item, so it will be possible to remove it via `AddMenu` component decorator.

## Examples

```tsx
<FileManagerViewConfig>
    {/* Grid config */}
    <Browser.Grid.Item.Thumbnail type={"*/*"} element={<DefaultRenderer />} />
    <Browser.Grid.Item.Action name={"download"} element={<GridDownload />} />
    {/* File details config */}
    <FileDetails.Preview.Thumbnail type={"*/*"} element={<DefaultRenderer />} />
    <FileDetails.Action name={"copyUrl"} element={<FileDetailsCopyUrl />} />
</FileManagerViewConfig>
```

## How Has This Been Tested?
Manually.

## Documentation
Pending. Need to update existing FileTypePlugin docs with this new API.

### CODE REVIEW

File: packages/app-file-manager/src/components/Grid/Thumbnail.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { useFile } from "~/hooks/useFile";
+import { useFileManagerViewConfig } from "~/modules/FileManagerRenderer/FileManagerView/FileManagerViewConfig";
+
+/**
+ * This component is used in the file grid to render thumbnails, as well as in the File Details preview.
```
Comment (adrians5j) [inline packages/app-file-manager/src/components/Grid/Thumbnail.tsx]: Is this comment relevant? 🤔

File: packages/app-file-manager/src/components/FileDetails/components/Thumbnail.tsx [hunk 2]
```diff
@@ -1,19 +1,15 @@
 import React from "react";
-import getFileTypePlugin from "~/getFileTypePlugin";
 import { useFile } from "~/hooks/useFile";
+import { useFileManagerViewConfig } from "~/modules/FileManagerRenderer/FileManagerView/FileManagerViewConfig";
 
 /**
  * This component is used in the file grid to render thumbnails, as well as in the File Details preview.
```
Comment (adrians5j) [inline packages/app-file-manager/src/components/FileDetails/components/Thumbnail.tsx]: Same question.

**Review by adrians5j — APPROVED** [review]
LGTM, just a question re a potentially outdated comment. 🤔

**Comment by Pavel910** [comment]
> LGTM, just a question re a potentially outdated comment. 🤔

Removed the outdated comments, they're no longer needed.

**Pull request merged**